File: VB/DXChartVizualizer.Test/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports System.Data.OleDb
Imports System.Collections

Namespace DXChartVizualizer.Test
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            Dim connection As New OleDbConnection("Provider=Microsoft.Jet.OLEDB.4.0;Data Source='C:\Program Files\Developer Express .NET v8.2\Demos\Data\nwind.mdb'")
            Dim selectCommand As New OleDbCommand("SELECT * FROM Categories")
            Dim da As New OleDbDataAdapter(selectCommand)
            Dim ds As New DataSet()

            selectCommand.Connection = connection
            da.Fill(ds, "Categories")

            selectCommand.CommandText = "SELECT TOP 10 * FROM Products"
            da.Fill(ds, "Products")

            dataGridView1.DataSource = ds
            dataGridView1.DataMember = ds.Tables(0).TableName

            Dim ds2 As New DataSet()
            Dim dt As New DataTable("MyDataTable")

            dt.Columns.Add("MyRow", GetType(String))
            dt.Columns.Add("MyData", GetType(Double))

            dt.Rows.Add(New Object() { "A", 103 })
            dt.Rows.Add(New Object() { "B", 200 })

            ds2.Tables.Add(dt)

            ' You can inspect Visualizer for the different DataTables represented
            ' in the variables "ds", "ds2" above.
            System.Diagnostics.Debugger.Break()
        End Sub
    End Class
End Namespace

File: CS/DXChartVizualizer.Test/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using System.Data.OleDb;
using System.Collections;

namespace DXChartVizualizer.Test
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            OleDbConnection connection = new OleDbConnection(@"Provider=Microsoft.Jet.OLEDB.4.0;Data Source='C:\Program Files\Developer Express .NET v8.2\Demos\Data\nwind.mdb'");
            OleDbCommand selectCommand = new OleDbCommand("SELECT * FROM Categories");
            OleDbDataAdapter da = new OleDbDataAdapter(selectCommand);
            DataSet ds = new DataSet();

            selectCommand.Connection = connection;
            da.Fill(ds, "Categories");

            selectCommand.CommandText = "SELECT TOP 10 * FROM Products";
            da.Fill(ds, "Products");

            dataGridView1.DataSource = ds;
            dataGridView1.DataMember = ds.Tables[0].TableName;

            DataSet ds2 = new DataSet();
            DataTable dt = new DataTable("MyDataTable");

            dt.Columns.Add("MyRow", typeof(string));
            dt.Columns.Add("MyData", typeof(double));

            dt.Rows.Add(new object[] { "A", 103 });
            dt.Rows.Add(new object[] { "B", 200 });

            ds2.Tables.Add(dt);

            // You can inspect Visualizer for the different DataTables represented
            // in the variables "ds", "ds2" above.
            System.Diagnostics.Debugger.Break();
        }
    }
}